lib/models/PPEStepInfo.dart

```diff
@@ -1,10 +1,7 @@
-import 'package:flutter/material.dart';
-
 class PPEStepInfo {
-  final String step;
-  final Image image;
+  final String image;
   final String text;
   final List<String> notes;
 
-  PPEStepInfo({this.step, this.image, this.text, this.notes});
+  const PPEStepInfo({this.image, this.text, this.notes});
```

Comment:
all 3 params are needed, so using optional value may result in missing data. You can either add @required or use mandatory params here

Reply:
Resolved in b890370